src/main/java/seedu/duke/Parser.java

```diff
@@ -13,24 +13,18 @@ public static Command parse(String command) {
 
         if (command.startsWith("new o/NUSNet")) {
             return new AddNUSNetCommand(command);
-        }
-        else if (command.startsWith("new o/StudentID")) {
```

Comment:
Can the functions for adding passwords be separated if not this list will be very very long